client/index.html

```diff
@@ -9,6 +9,7 @@
 	<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 	<meta name="mobile-web-app-capable" content="yes">
 	<meta name="referrer" content="no-referrer">
+	<meta name="theme-color" content="#455164">
```

Comment:
You're talking about the background color, I'm talking about the toolbar color.

It's not as bad as a website because when set up as an app, the URL + icons bar disappears, but this is not true for users that did not set up The Lounge as a standalone app (such as those testing the demo, or pretty much all public instances). Also applies to the very top bar, where time and battery/connectivity icons live, which is affected by the `theme-color`.